### PR TITLE
Remove usernames and emails from AI traces.

### PR DESCRIPTION
**Pull request by cristinamanum**
Remove usernames and emails from AI traces.

### CODE REVIEW

File: src/NuGetGallery/Authentication/AuthenticationService.cs [hunk 1]
```diff
@@ -756,7 +755,7 @@ private User FindByUserNameOrEmail(string userNameOrEmail)
                 else
                 {
                     // If multiple matches, leave it null to signal no unique email address
-                    _trace.Warning("Multiple user accounts with email address: " + userNameOrEmail + " found: " + String.Join(", ", allMatches.Select(u => u.Username)));
+                    _trace.Warning("Multiple user accounts with a single email address were found. Count: " + String.Join(", ", allMatches.Select(u => u.Username)));
```
Comment (loic-sharma) [inline src/NuGetGallery/Authentication/AuthenticationService.cs]: I'm guessing `String.Join(", ", allMatches.Select(u => u.Username))` should be `allMatches.Count()`?

**Comment by chenriksson** [comment]
SecurePushSubscription may have `_diagnostics.Information` including username.

Client IP is ok in AI logs, right?

File: src/NuGetGallery/Authentication/AuthenticationService.cs (hunk 1 above)
Comment (cristinamanum) [reply src/NuGetGallery/Authentication/AuthenticationService.cs]: Right, good catch. Fixed.

Comment (cristinamanum) [reply src/NuGetGallery/Authentication/AuthenticationService.cs]: @chenriksson - First comment - thank you - fixed. 
Second comment - plain Client IP is not ok - needs to be obfuscated.

Comment (cristinamanum) [reply src/NuGetGallery/Authentication/AuthenticationService.cs]: Verified the traces for cookie compliance and do not seem to be IPs in the messages.

**Pull request merged**